Source/JavaScriptCore/runtime/GenericTypedArrayView.h

```diff
@@ -40,6 +40,7 @@ class GenericTypedArrayView final : public ArrayBufferView {
     static RefPtr<GenericTypedArrayView> tryCreate(size_t length);
     static RefPtr<GenericTypedArrayView> tryCreate(const typename Adaptor::Type* array, size_t length);
     static RefPtr<GenericTypedArrayView> tryCreate(std::span<const typename Adaptor::Type> data) { return tryCreate(data.data(), data.size()); }
+    static RefPtr<GenericTypedArrayView> tryCreate(Ref<ArrayBuffer>&&);
```

Comment:
Since this never returns `nullptr`, this could be `create` instead of `tryCreate`, and return a `Ref` instead of `RefPtr`.